examples/trigger-phrase/agent.py

```diff
@@ -0,0 +1,166 @@
+import asyncio
+import logging
+from typing import AsyncIterable, Optional
+
+from dotenv import load_dotenv
+from livekit import rtc
+from livekit.agents import (
+    AutoSubscribe,
+    JobContext,
+    WorkerOptions,
+    cli,
+    stt,
+    tokenize,
+    tts,
+    llm,
+)
+from livekit.plugins import deepgram, openai, silero, elevenlabs
+
+load_dotenv()
+
+logger = logging.getLogger("my-worker")
+logger.setLevel(logging.INFO)
+
+word_tokenizer_without_punctuation: tokenize.WordTokenizer = (
+    tokenize.basic.WordTokenizer(ignore_punctuation=True)
+)
+
+trigger_phrase = "Hi Bob!"
```

Comment:
nit: `TRIGGER_PHRASE` instead to show that this is a changeable constant

Reply:
oh okay, didn't know that - thanks!